src/lib.rs

```diff
@@ -123,6 +124,12 @@ impl DisplayStr for &str {
         f.str(self)
     }
 }
+
+impl DisplayStr for Box<dyn DisplayStr> {
+    fn display_str(&self, f: &mut Formatter) -> Result<()> {
+        self.as_ref().display_str(f)
+    }
+}
 /// This trait is effectively an re-implementation for [std::fmt::Display]
```

Comment:
run `cargo fmt` before submit, it will add space between these.

Reply:
okay? Didn't happen to me, but ill review the files to ensure this.